custom_components/meross_cloud/cover.py

```diff
@@ -130,6 +130,10 @@ def supported_features(self):
         # So we don't support that on HA neither.
         return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP
 
+    @property
+    def current_cover_position(self):
+        return self._device.get_position(channel=self._channel_id) or 50
```

Comment:
Why assuming 50 if the underlying library reposts None? Returning None should be OK with HA, it will determine "unknown". Can you please try that way and report if that's working as intended?

Reply:
`None` is treated as closed and `up` button is disabled, at least with `50` both buttons are enabled

Reply:
I tried to sniff data but I can't find like a initial position information after restarting home assistant. The data is collected properly either for open, close and/or stop operations, but for getting initial position after restarting I'm not able to see any. Meros APP does it whenever it is launched

Reply:
Hi!
Are you using the sniffer tool I linked in the documentation? I need the data collected by the sniffer in order to align the low-level library used by HomeAssistant. Please follow the steps described [here](https://albertogeniola.github.io/MerossIot/advanced-topics.html#sniff-device-data). 

When launching the sniffer make sure to operate the device from its physical buttons and also form the meross app. In this way Athe sniffer will capture inputs from the app and notifications from the device, and I can gather all the technical info needed to replicate such behavior in HA

Reply:
I've seen you have posted the sniffed data, that has been helpful. I believe I have found the issue with the low-level library affecting this specific case. I'll update the main thread when done with some instructions so that you can test the feature on my behalf with the updated MRS100

Reply:
looking forward for your update. I will be more than glad to test it

Reply:
after https://github.com/albertogeniola/meross-homeassistant/pull/495/commits/79b5362d4a3c048517ffd2290b4b917e56ae112d and https://github.com/albertogeniola/meross-homeassistant/pull/495/commits/80899b95ab81ae669db0e3e32a691b85e181c0ae initial position is properly retrieved.

Many thanks @albertogeniola 
I don't know whether `meross_iot` `0.4.7.0b1` is a final version, otherwise please let me know what's the one and I will adapt this PR